src/sage/rings/polynomial/polynomial_element.pyx

```diff
@@ -5211,13 +5211,13 @@ cdef class Polynomial(CommutativePolynomial):
             sage: # needs sage.rings.finite_rings
             sage: P.<x> = PolynomialRing(GF(401^13, 'a'))
             sage: t = 2*x^14 - 5 + 6*x
-            sage: t.splitting_field('b')
+            sage: t.splitting_field('b')                                                # long time -- 16s
```

Comment:
```suggestion
            sage: t.splitting_field('b')                                                # long time (16s)
```

Reply:
These details always go in parentheses. (The format matters because we have a tool that makes edits to these doctest tags in the source code.)